examples/cross-account/main.tf

```diff
@@ -0,0 +1,78 @@
+resource "aws_backup_global_settings" "aws_backup" {
```

Comment:
If possible don't use resource type in the resource name.

Reply:
renamed